packages/manager/apps/license/src/app.module.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable import/extensions */
+/* eslint-disable import/no-webpack-loader-syntax */
+import 'script-loader!moment/min/moment.min.js';
```

Comment:
`moment` should be a dev dependency and declared as `globals` in `eslintrc` file.